plugins/file/zipfile/zip_file_reader.go

```diff
@@ -0,0 +1,55 @@
+package zipfile
+
+import (
+	"archive/zip"
+	"bytes"
+	"io"
+
+	"github.com/chrislusf/gleam/util"
+)
+
+type FileReader struct {
+	reader   *zip.ReadCloser
+	Cursor   int
+	NumFiles int
+}
+
+func New(filename string) *FileReader {
+	r, _ := zip.OpenReader(filename)
+	numFilesAndDirs := len(r.File)
+	return &FileReader{
+		reader:   r,
+		Cursor:   0,
+		NumFiles: numFilesAndDirs,
+	}
+}
+
+func (r *FileReader) ReadHeader() (fieldNames []string, err error) {
+	return nil, nil
```

Comment:
set the header to "file_name", "file_content", since these fileds are added.

Reply:
this has been done